docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -0,0 +1,256 @@
+GeoMesa FSDS Quick Start
+========================
+
+This tutorial is the fastest and easiest way to get started with the
+GeoMesa FileSystem Datastore (FSDS). In the spirit of keeping things simple, the
+code in this tutorial only does a few things:
+
+1. Establishes a new (static) SimpleFeatureType
+2. Prepares the FSDS to store this type of data
+3. Creates a few thousand example SimpleFeatures
+4. Writes these SimpleFeatures to the FSDS
+5. Queries for a given geographic rectangle, time range, and attribute
+   filter, writing out the entries in the result set
+
+Prerequisites
+-------------
+
+Before you begin, you must have the following installed and configured:
+
+-  `Java <http://java.oracle.com/>`__ JDK 1.8
+-  Apache `Maven <http://maven.apache.org/>`__ |maven_version|
+-  a GitHub client
+
+Download and Build the Tutorial
+-------------------------------
+
+Pick a reasonable directory on your machine, and run:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/geomesa/geomesa-tutorials.git
+    $ cd geomesa-tutorials
+
+.. warning::
+
+    Make sure that you download or checkout the version of the tutorials project that corresponds to
+    your GeoMesa version. See :ref:`tutorial_versions` for more details.
+
+To ensure that the quick start works with your environment, modify the ``pom.xml``
+to set the appropriate versions for Hadoop, etc.
+
+For ease of use, the project builds a bundled artifact that contains all the required
+dependencies in a single JAR. To build, run:
+
+.. code-block:: bash
+
+    $ mvn clean install -pl geomesa-quickstart-fsds -am
+
+About this Tutorial
+-------------------
+
+The quick start operates by inserting and then querying several thousand features.
+After the insertions are complete, a sequence of queries are run to
+demonstrate different types of queries possible via the GeoTools API.
+
+Running the Tutorial
+--------------------
+
+On the command line, run:
+
+.. code-block:: bash
+
+    $ java -cp geomesa-quickstart-fsds/target/geomesa-quickstart-fsds-$VERSION.jar \
+        org.geomesa.example.fsds.FSDSQuickStart \
+        --fs.path /tmp/fsds/                    \
+        --fs.encoding parquet
+
+The arguments above indicate:
+
+-  ``fs.path`` the FSDS storage path. Each simple feature type will produce a sub-directory here.
+-  ``fs.encoding`` the file storage encoding. The quickstart comes pre-configured to use Apache's Parquet encoding.
+
+Once run, you should see the following output:
+
+.. code-block:: none
+
+    Loading datastore
+    Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
```

Comment:
we can remove this line, even though it gets printed out... looks kind of bad here :)